content/updates/governance/3-Revamped-yes-no-attribute.md

```diff
@@ -0,0 +1,24 @@
+# Meet the new Yes/No attribute
+
+## New Yes/No attribute in the product form
+::: meta-data type="Improvement" ee-only="false"
+
+The Yes/No attribute is now more user-friendly: you can easily see if your Yes/No attribute has no value yet. You can also empty it if needed.
+
+![New Yes/No attribute design](../img/new-yes-no-attribute-design.png)
+
+## Better filter with the Yes/No attribute
+::: meta-data type="New" ee-only="false"
+
+Two new operators `is empty` and `is not empty` are now available to help you filter on Yes/No attributes in the product grid and to automate enrichment with the rules engine. These new filters help you boost your productivity and be even more efficient when looking for specific products and product models.
```

Comment:
```suggestion
Two new operators `is empty` and `is not empty` are now available in the product grid and in the Rules Builder, to help you filter on Yes/No attributes. These new filters will help you to easily find what you were looking for in the product grid and be even more precise while building your rules.
```
Can we put a link on Rules Builder to link it to the relevant section in Automation?

Reply:
```suggestion
Two new operators `is empty` and `is not empty` are now available in the product grid and in the [Rules Builder](TODO), to help you filter on Yes/No attributes. These new filters will help you to easily find what you were looking for in the product grid and be even more precise while building your rules.
```